BrainPortal/app/models/boutiques_portal_task.rb

```diff
@@ -267,6 +267,12 @@ def final_task_list #:nodoc:
     self.addlog(descriptor.file_revision_info.format("%f rev. %s %a %d"))
     valid_input_keys = descriptor.inputs.map(&:id)
 
+    # Add information about Boutiques module
+    module_information = boutiques_module_information()
```

Comment:
Can we simplify this even further now? Why assign the information to a variable?